Web_Development/Javascript/Git and Bash/readme.md

```diff
@@ -0,0 +1,152 @@
+### Git and Bash
+Git and Bash are fairly different things but windows version of `git` comes with configured with bash shell. This is why  you can even execute the bash commands their . But you can not perform low level tasks with git-bash in windows.
```

Comment:
There is a typo in the second sentence. Also, remove extra spaces wherever present.
> This is why you can even execute the bash commands ~their~ there.

Reply:
I have fixed that sir .